### PR TITLE
using SafeERC20 to implement safeTransfer in Crowdsale

### PR DESCRIPTION
**Pull request by dougiebuckets**
Fixes #985 

# 🚀 Description

Introduced safeTransfer to Crowdsale. Opened new pull request b/c I accidently closed this one: https://github.com/OpenZeppelin/openzeppelin-solidity/pull/985

- [x] 📘 I've reviewed the [OpenZeppelin Contributor Guidelines](../blob/master/CONTRIBUTING.md)
- [x] 🎨 I've run the JS/Solidity linters and fixed any issues (`npm run lint:all:fix`).

### CODE REVIEW

File: contracts/crowdsale/Crowdsale.sol [hunk 1]
```diff
@@ -2,6 +2,8 @@ pragma solidity ^0.4.24;
 
 import "../token/ERC20/ERC20.sol";
 import "../math/SafeMath.sol";
+import "../token/ERC20/ERC20Basic.sol";
```
Comment (frangio) [inline contracts/crowdsale/Crowdsale.sol]: This new `import` is not necessary.

**Comment by elopio** [comment]
One more thing @dougiebuckets. There is a file  .node-xmlhttprequest-sync-7601, can you please remove it?

**Comment by dougiebuckets** [comment]
Good catch, @elopio. Thank you. Update made.

**Comment by frangio** [comment]
Thanks @dougiebuckets!

**Pull request merged**